### ISSUE TITLE
Group flags wiki

### ISSUE THREAD

**Issue by Vlader97** [Zrips/Residence#447]
I often solve problems with res flags. Some of them are grouped, but I do not know which all the flags contain. I would like to see in the wiki which flags are grouped and which all flags contain.

For example:

**use** 
- button
- door
- level
- etc…

**Comment by Zrips** [comment]
I think you are talking about which materials are affected by specific flags and not grouped flags, which can be found at flags.yml and at GroupedFlags section.

**Comment by Vlader97** [comment]
Yes describe which all materials or interactions are affected by specific flag and if override any flag. This would help me prevent set one thing by two flags.
I would put flags that override a specific flag in the list below it. For example.

**use** – Allows or denys use of doors, levers, buttons, trapdoors, (what all ?)
- **button** – Allows or denys players to use buttons, overrides use.
- **door** – Allows or denys players to use doors and trapdoors, overrides use.
- **lever** – Allows or denys players to use levers, overrides use.
